src/graph/graph.c

```diff
@@ -83,16 +83,6 @@ bool _select_op_free_edge(GrB_Index i, GrB_Index j, GrB_Index nrows, GrB_Index n
 
 /* ========================= Synchronization functions ========================= */
 
-/* Acquire mutex when a reader thread may modify shared data. */
```

Comment:
Generally speaking, did you thought of a test which will try and trigger this issue?